plugins/c8y_firmware_plugin/src/main.rs

```diff
@@ -21,7 +26,16 @@ use tedge_config::MqttPortSetting;
 use tedge_config::TEdgeConfig;
 use tedge_config::TmpPathSetting;
 use tedge_config::DEFAULT_TEDGE_CONFIG_PATH;
+use tedge_utils::file::create_directory_with_user_group;
 use tokio::sync::Mutex;
+use tracing::info;
+
+// TODO! We should make it configurable by tedge config later.
+const PERSISTENT_DIR_PATH: &str = "/var/tedge";
+
+pub const CACHE_DIR_NAME: &str = "cache";
+pub const FILE_TRANSFER_DIR_NAME: &str = "file-transfer";
```

Comment:
TODO for later: This constant ideally should have been defined in some `tedge-api` crate so that all plugins using the file-transfer functionality could reuse it.